app/controllers/poems_controller.rb

```diff
@@ -60,6 +60,8 @@ def notify_to_slack
 #{current_user.name}
     EOC
 
-    Slack.chat_postMessage text: text, username: "PoemMaster", channel: ENV['SLACK_CHANNEL']
+    if ENV['SLACK_CHANNEL'].present?
```

Comment:
__[rubocop]__
* Use a guard clause instead of wrapping the code inside a conditional expression. [:ref](https://github.com/bbatsov/ruby-style-guide#no-nested-conditionals)